app/controllers/reactions_controller.rb

```diff
@@ -130,4 +140,13 @@ def rate_article(reaction)
                       context: "readinglist_reaction",
                       rating: current_user.experience_level)
   end
+
+  def clear_article_reactions(id, type, mod, category)
+    reactions = if category == "thumbsup"
+                  Reaction.where(reactable_id: id, reactable_type: type, user: mod).where.not(category: category)
+                elsif %w[thumbsdown vomit].include?(category)
```

Comment:
We often follow a pattern of assigning things like this array to constants. Otherwise, we have to re-create the array and two strings on each method invocation. For example:

```ruby
NONSENSICAL_PAIRS = %w[thumbsdown vomit].freeze
...
elsif category.in?(NONSENSICAL_PAIRS)
```